### PR TITLE
Mint signup clone feature a

### PR DESCRIPTION
**Pull request by KossySteve**
This is a clone of mint.com Intuit sign up page

Features:
-Added navigation bar
-Added HTML form with required elements
-Added footer and required text and dummy links

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,9 +1,58 @@
...
+	<header>
+		<nav>
+			<button id="nav_signin_button">Sign in</button>
+			<ul id="nav_instructions">
+				<li>Already have an Intuit account?</li>
+				<li><a href="#">I forgot my User ID or Password</a></li>
+			</ul>
+		</nav>
+	</header>
+	<section id="container">
```
Comment (addod19) [inline index.html]: Let improve semantics by using ```<main>```  instead of ``` <section id="container">```

**Review by addod19 — CHANGES_REQUESTED** [review]
## Status: Require changes 🔴

Hi @KossySteve   and @Elbie-em 😊
The project looks great. README looks professional, descriptive commits etc.
There there are few things we have to do to get this project approved check comment below : 😊
- Kindly enable stickler for the repository, we have the stickler.yml file but we need to enable for the repo. Kindly check this [link](https://stickler-ci.com/) to enable stickler.

[ Optional ]
- I would recommend you use ```css``` for all your styling, unless extreme case you use ``id```.

Kindly make these changes and submit project for review
Happy coding.

[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo-9738b0128/)

**Review by pelzolga123 — CHANGES_REQUESTED** [review]
Project requires changes.

**Comment by pelzolga123** [comment]
## Hi Team!  
### Your project looks :+1: amazing!
Here what you need to fix, before this project can be approved:
* [ ] Check HTML markup with this [validator](https://validator.w3.org/#validate_by_upload). Now it shows errors in your project.
* [ ] You added .stickler.yml file but you haven't turned on stickler for this repo. Please follow [this link](https://stickler-ci.com/) to turn on stickler for this repo. Full instructions on setting up stickler, can be found [here](https://github.com/microverseinc/linters-config/tree/master/css)
* [ ] Input `phone_number` should have type `tel` ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/html_forms/form_and_input_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59524927/))

### Please kindly fix the above issues and send another code review request.

If you have any questions or need help, you can message me on slack (@ Olga Pelts). *Happy coding*

**Review by pelzolga123 — APPROVED** [review]
Approved.

**Comment by pelzolga123** [comment]
## Great  work, Congratulation :+1: Project Approved.
You did all changes required previously. Go ahead and merge to master.
_Goodluck in your next project_ *Happy coding*

**Pull request merged**